xds/src/main/java/io/grpc/xds/ClientXdsClient.java

```diff
@@ -164,6 +164,8 @@
       "type.googleapis.com/envoy.extensions.clusters.aggregate.v3.ClusterConfig";
   private static final String TYPE_URL_TYPED_STRUCT =
       "type.googleapis.com/udpa.type.v1.TypedStruct";
+  private static final String NEW_TYPE_URL_TYPED_STRUCT =
```

Comment:
"NEW" tends to be confusing rather rapidly in the future. How about "XDS" instead of "NEW"? You can also rename the previous type to "UDPA". Another option is to use "TYPE_URL_TYPED_STRUCT" for the xds version, and _only_ call the udpa one "UDPA" or "OLD".